code/modules/mob/mob_helpers.dm

```diff
@@ -968,4 +979,127 @@ proc/is_blind(A)
 	P.time_of_death[which] = value
 	return 1
 
+
+//Below here is stuff related to devouring, but which is generally helpful and thus placed here
+//See Devour.dm for more info in how these are used
+
+
+//Flags for the eat_types variable, a bitfield of what can or can't be eaten
+//Note that any given mob can be more than one type
+#define TYPE_ORGANIC	1//Almost any creature under /mob/living/carbon and most simple animals
+#define	TYPE_SYNTHETIC	2//Everything under /mob/living/silicon, plus IPCs, viscerators
+#define TYPE_HUMANOID	4//Humans, skrell, unathi, tajara, vaurca, diona, IPC, vox
+#define TYPE_WIERD		8//Slimes, constructs, demons, and other creatures of a magical or bluespace nature.
+
+
+//Blacklists of mobs that can be excluded from eating by flags in the bitfield
+var/list/humanoid_mobs_specific = list( /mob/living/carbon/human,
+	/mob/living/carbon/human/bst,
+	/mob/living/carbon/human/skrell,
+	/mob/living/carbon/human/unathi,
+	/mob/living/carbon/human/diona,
+	/mob/living/carbon/human/tajaran,
+	/mob/living/carbon/human/vox,
+	/mob/living/carbon/human/machine,
+	/mob/living/carbon/human/bug
+
+	)
+
+var/list/humanoid_mobs_inclusive = list(
+	/mob/living/simple_animal/hostile/pirate,
+	/mob/living/simple_animal/hostile/russian,
+	/mob/living/simple_animal/hostile/syndicate
+	)
+
+var/list/synthetic_mobs_specific = list(
+	/mob/living/carbon/human/machine,
+	/mob/living/simple_animal/hostile/retaliate/malf_drone,
+	/mob/living/simple_animal/hostile/viscerator,
+	/mob/living/simple_animal/spiderbot
+	)
+
+
+var/list/synthetic_mobs_inclusive = list( /mob/living/silicon,
+	/mob/living/simple_animal/hostile/hivebot,
+	/mob/living/bot
+	)
+
+var/list/wierd_mobs_specific = list(/mob/living/simple_animal/adultslime)
+
+var/list/wierd_mobs_inclusive = list( /mob/living/simple_animal/construct,
+	/mob/living/simple_animal/shade,
+	/mob/living/simple_animal/slime,
+	/mob/living/simple_animal/hostile/faithless,
+	/mob/living/carbon/slime
+	)
+
+
+/mob/living/proc/find_type()
+	//This function returns a bitfield indicating what type(s) the passed mob is.
+	//Synthetic and wierd are exclusive from organic. We assume it's organic if it's not either of those
+	var/mob/living/test = src

```

Comment:
This is unnecessary. You should be able to replace all instances of `test` with `src`.